migrator/runner/version.go

```diff
@@ -61,13 +63,28 @@ func GetCurrentSeqNumRocksDB(db *gorocksdb.DB) (int, error) {
 	return int(version.GetSeqNum()), nil
 }
 
+func getCurrentSeqNumPostgres(databases *types.Databases) (int, error) {
+	migVer, err := migrations.ReadVersionPostgres(databases.PostgresDB)
+	if err != nil {
+		return 0, errors.Wrap(err, "getting current postgres sequence number")
+	}
+
+	return migVer.SeqNum, nil
+}
 func getCurrentSeqNum(databases *types.Databases) (int, error) {
+	// If Rocks and Bolt are passed into this function when Postgres is enabled, that means
+	// we are in a state where we need to migrate Rocks to Postgres.  In this case the Rocks
+	// sequence number will take precedence and drive the migrations
+	if features.PostgresDatastore.Enabled() && databases.RocksDB == nil && databases.BoltDB == nil {
```

Comment:
it is not that simple.  if we are upgrading from Rocks -> Postgres we need to get the sequence number from Rocks to start the migration process.  The Postgres sequence won't get set until we start going through n01.  So I still need the other check to make sure I only get the postgres sequence when I need it.  The clone manager will only send all databases to the runner if it determines it needs to go from Rocks -> Postgres.  Otherwise it will only send one or the other.  So we shouldn't get in this situation.

Reply:
I think I covered your intent with the latest update

Reply:
Right. Got you